gui/Cargo.lock

```diff
@@ -239,9 +239,9 @@ checksum = "9e1b586273c5702936fe7b7d6896644d8be71e6314cfe09d3167c95f712589e8"
 
 [[package]]
 name = "base64"
-version = "0.21.0"
+version = "0.21.6"
```

Comment:
This commit says it bumps the Liana version but it doesn't.

Reply:
Yes, my bad. I need to update it